Source/JavaScriptCore/b3/air/AirOpcode.opcodes

```diff
@@ -743,7 +743,7 @@ MoveDouble U:F:64, D:F:64
 MoveDouble U:F:64, D:F:64, S:F:64
     Addr, Addr, Tmp
 
-MoveVector U:F:128, D:F:128
+64: MoveVector U:F:128, D:F:128
```

Comment:
These end up excluding RISCV64 until RISCV64 also starts being included under that condition since it's a 64-bit ISA. These can be avoided by using the macro to produce the necessary no-ops in the MacroAssemblerRISCV64 class.